lib/samples/find_route/find_route_sample.dart

```diff
@@ -25,20 +25,31 @@ class FindRouteSample extends StatefulWidget {
 }
 
 class _FindRouteSampleState extends State<FindRouteSample> {
+  // the map view controller.
   final _mapViewController = ArcGISMapView.createController();
+  // the graphics overlay for the stops.
   final _stopsGraphicsOverlay = GraphicsOverlay();
+  // the graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
+  // the symbol for the route line.
   final _routeLineSymbol = SimpleLineSymbol(
       style: SimpleLineSymbolStyle.solid, color: Colors.blue, width: 5.0);
+  // the symbol for the start and end stops.
   final _routeStartSymbol = SimpleMarkerSymbol(
       style: SimpleMarkerSymbolStyle.circle, color: Colors.green, size: 8.0);
   final _routeEndSymbol = SimpleMarkerSymbol(
       style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 8.0);
```

Comment:
`_routeStartSymbol` and `_routeEndSymbol` are only ever used one time in one place, so it's weird to see them as member variables. I think they should just be created where they are used, down in `initStops()`.

Reply:
Just like `_routeLineSymbol`, should I also move it to generateRoute()?

Reply:
yeah that makes sense